.travis.yml

```diff
@@ -27,16 +27,12 @@ before_install:
       npm install -g codecov
     fi
 install:
-  - pip install --upgrade -e ".[test]"
-  - |
-    if [[ $GROUP == js ]]; then
-      cd nbdime-web
-      npm install
```

Comment:
PS: This should actually only be removed if we keep the develop install, otherwise we need it.